src/Proto.Persistence/Persistence.cs

```diff
@@ -28,7 +28,7 @@ public void Init(IProvider provider, IContext context)
             if (t != null)
             {
                 EventIndex = t.Item2;
-                Context.ReceiveAsync(t.Item1).Wait();
+                await Context.ReceiveAsync(t.Item1);

```

Comment:
hmm, this is what is in the PR:

![change](https://cloud.githubusercontent.com/assets/647031/23315951/bae35e7a-fac8-11e6-9f35-289d04c80317.png)

Oh well, Ill merge, if it turns out strange we can always fix it :-)